resources/assets/js/label-trees/components/labelTreeLabel.vue

```diff
@@ -13,6 +13,10 @@
             </span>
             <span class="label-tree-label__buttons">
                 <button v-if="showFavourites" type="button" class="label-tree-label__favourite" :class="favouriteClass" @click.stop="toggleFavourite" :title="favouriteTitle">
+                    <span v-if="showFavouriteShortcuts" @click.stop="!toggleFavourite" >
+                        <span class="fa fa-keyboard" aria-hidden="true" title=""></span>
+                        <span v-text="addOnePosition()"></span>
```

Comment:
`addOnePosition` is a computed property. These are not called like methods. Here you would write `:text="addOnePosition"`. In the code you would write `this.addOnePosition`. Since a computed property is used like a static property, you would also name it differently (not like a method), e.g. `positionText` or `actualPosition`.